pyminion/game.py

```diff
@@ -115,9 +116,9 @@ def _create_basic_piles(self) -> List[Pile]:
                 basic_piles.append(Pile([card] * SILVER_LENGTH))
             elif card.name == "Gold":
                 basic_piles.append(Pile([card] * GOLD_LENGTH))
-            elif "Victory" in card.type:
+            elif CardType.Victory in card.type:
                 basic_piles.append(Pile([card] * VICTORY_LENGTH))
-            elif card.name == "Curse":
```

Comment:
This as well should remain a string. Or better yet, copy the above victory card check. 

```python
elif CardType.Curse in card.type:
                basic_piles.append(Pile([card] * CURSE_LENGTH))
```